runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/ResettableThrowingStreamObserver.java

```diff
@@ -104,6 +104,10 @@ synchronized void poison() {
     }
   }
 
+  synchronized boolean hasReceivedPoisonPill() {
```

Comment:
nit: how about isPoisoned?

Poison pill was a special element we added to queues etc, it's a little confusing to use the term here to me.

Reply:
removed